.eslintrc.json

```diff
@@ -2,7 +2,8 @@
   "root": true,
   "extends": ["./node_modules/sanctuary-style/eslint-es3.json"],
   "parserOptions": {
-    "sourceType": "module"
+    "sourceType": "module",
+    "ecmaVersion": 3
```

Comment:
@Amri91 and I found that this doesn't currently work in eslint. Did you get it to work?

Reply:
No, I am looking for a workaround.